### PR TITLE
fixes/enhancements to the psnr test

### PR DESCRIPTION
**Pull request by bbaldino**
SQUASH BEFORE MERGING

### CODE REVIEW

File: scripts/psnr-test.sh [hunk 1]
```diff
@@ -16,11 +16,20 @@
...
-INPUT_FRAME_DIR=$2
-RESIZED_FRAME_DIR=$3
+OUTPUT_FRAME="$1"
+INPUT_FRAME_DIR="$2"
+RESIZED_FRAME_DIR="$3"
+QR_CODE_DECODER="zbarimg"
 
-FRAME_NUMBER=$(java -jar $SCRIPT_DIR/javase-3.2.2-SNAPSHOT-jar-with-dependencies.jar $OUTPUT_FRAME |head -3|tail -1)
+
+PATH_TO_QR_DECODER=$(which $QR_CODE_DECODER)
```
Comment (gpolitis) [inline scripts/psnr-test.sh]: would it make sense to leave the old decoder in a comment for future reference?

File: scripts/push_psnr_results.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python
```
Comment (gpolitis) [inline scripts/push_psnr_results.py]: Adding the licence header and a brief description of what this does would be nice

File: scripts/psnr-test.sh (hunk 1 above)
Comment (bbaldino) [reply scripts/psnr-test.sh]: i figured the git history would be enough (i did still leave the jar in the repo).  do you think i should add the old one back and leave it commented?

File: scripts/push_psnr_results.py (hunk 2 above)
Comment (bbaldino) [reply scripts/push_psnr_results.py]: good idea, will do

File: scripts/psnr-test.sh (hunk 1 above)
Comment (gpolitis) [reply scripts/psnr-test.sh]: Actually feel free to disregard my comment. I suppose you have tested with zbarimg and you got no false negatives thus I don't see why we would ever want to go back to ZXing.

**Comment by gpolitis** [comment]
LGTM. I'll wait a couple of hours in case anybody else wants to comment on it and then I'll merge it

File: src/test/java/org/jitsi/meet/test/PSNRTest.java [hunk 3]
```diff
@@ -196,6 +207,20 @@ public void testPSNR()
                 File outputFrameFile = new File(outputFrame);
                 outputFrameFile.delete();
             }
+            float averagePsnr = totalPsnr / framesCount;
```
Comment (bgrozev) [inline src/test/java/org/jitsi/meet/test/PSNRTest.java]: Would we also want to fail the test if the average is below a certain threshold (the per-frame threshold is very very low)?

File: scripts/psnr-test.sh [hunk 4]
```diff
@@ -16,11 +16,20 @@
...
 SCRIPT_DIR=$(dirname $0)
-OUTPUT_FRAME=$1
-INPUT_FRAME_DIR=$2
-RESIZED_FRAME_DIR=$3
+OUTPUT_FRAME="$1"
+INPUT_FRAME_DIR="$2"
+RESIZED_FRAME_DIR="$3"
+QR_CODE_DECODER="zbarimg"
 
-FRAME_NUMBER=$(java -jar $SCRIPT_DIR/javase-3.2.2-SNAPSHOT-jar-with-dependencies.jar $OUTPUT_FRAME |head -3|tail -1)
```
Comment (bgrozev) [inline scripts/psnr-test.sh]: Do we still need that jar if we're using zbarimg instead?

File: src/test/java/org/jitsi/meet/test/PSNRTest.java (hunk 3 above)
Comment (bbaldino) [reply src/test/java/org/jitsi/meet/test/PSNRTest.java]: we could, but i think the per-frame one will be even more sensitive, no? (unless you mean having the average required to be an even higher value?)

File: scripts/psnr-test.sh (hunk 4 above)
Comment (bbaldino) [reply scripts/psnr-test.sh]: no...and i think you're right, we should remove it.  i'll delete it.

**Comment by bbaldino** [comment]
@gpolitis all good here?

**Pull request merged**